webpack/scenes/ContentViews/Details/Repositories/ContentViewRepositories.js

```diff
@@ -324,7 +291,62 @@ const ContentViewRepositories = ({ cvId, details }) => {
           }
         </Split>
       }
-    />
+    >
+      <Thead>
+        <Tr key="version-header">
+          <Th key="select-all" />
+          {columnHeaders.map((title, index) => {
+            if (index === 0) {
+              return <Th modifier="fitContent" key={`col-header-${title}`}>{title}</Th>;
+            }
+            return <Th key={`col-header-${title}`}>{title}</Th>;
+          })}
+        </Tr>
+      </Thead>
+      <Tbody>
+        {results?.map((repo) => {
+          const {
+            id,
+            content_type: contentType,
+            name,
+            added_to_content_view: addedToCV,
+            product: { id: productId, name: productName },
+            content_counts: counts,
+            last_sync_words: lastSyncWords,
+            last_sync: lastSync,
+          } = repo;
+          return (
+            <Tr key={id}>
+              <Td>
+                <Checkbox
+                  id={id}
+                  isChecked={isSelected(id)}
+                  onChange={selected =>
+                    selectOne(selected, id, repo)
+              }
+                />
+              </Td>
```

Comment:
This column needs to be conditional (column header as well), on permissions
![Screen Shot 2022-03-28 at 1 06 36 PM](https://user-images.githubusercontent.com/38083295/160469062-ff8f5bf2-b09d-4309-ad10-767834594f18.png)
.

Reply:
The above was viewed with "viewer" permissions only.